test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKRuntimeOS.java

```diff
@@ -12,37 +12,44 @@
 
 /**
  * CDK runtime OS enum types
+ * 
  * @author odockal
  *
  */
 public enum CDKRuntimeOS {
 
-	LINUX("linux", ""),
-	WINDOWS("win", ".exe"),
-	MAC("darwin", ""),
+	LINUX("linux", "linux", ""), 
```

Comment:
@odockal maybe there could comment what these strings means...

Reply:
updated in class javadoc.